### PR TITLE
[android/tflite] Add tflite v2.7.0 support

### PR DESCRIPTION
**Pull request by anyj0527**
- Add tflite v2.7.0 support
- Set v2.7.0 as default version of tflite

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

### CODE REVIEW

File: java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk [hunk 1]
```diff
@@ -37,13 +37,18 @@ TFLITE_FLAGS := \
...
 TFLITE_ENABLE_XNNPACK_DELEGATE := false
+TFLITE_REQUIRE_XNNPACK_PREBUILT_LIBS := false
 TFLITE_EXPORT_LDLIBS :=
 
 ifeq ($(shell test $(TFLITE_VERSION_MAJOR) -ge 2; echo $$?),0)
 TFLITE_ENABLE_GPU_DELEGATE := true
 TFLITE_ENABLE_NNAPI_DELEGATE := true
 TFLITE_ENABLE_XNNPACK_DELEGATE := true
 
+ifeq ($(TFLITE_VERSION_MINOR),3)
```
Comment (jaeyun-jung) [inline java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk:48]: let's change cond (tflite_ver == 2.3.0, major ==2 and minor == 3)

File: java/build-nnstreamer-android.sh [hunk 2]
```diff
@@ -122,9 +122,9 @@ flatbuf_ver="1.12.0"
 enable_mqtt="no"
 paho_mqtt_c_ver="1.3.7"
 
-# Set tensorflow-lite version (available: 1.9.0 / 1.13.1 / 1.15.2 / 2.3.0)
-tf_lite_ver="2.3.0"
-tf_lite_vers_support="1.9.0 1.13.1 1.15.2 2.3.0"
+# Set tensorflow-lite version (available: 1.9.0 / 1.13.1 / 1.15.2 / 2.3.0 / 2.7.0)
+tf_lite_ver="2.7.0"
```
Comment (jaeyun-jung) [inline java/build-nnstreamer-android.sh:126]: You should fix line 519 as default version in .mk file is changed.
(sed -i "s|TFLITE_VERSION := 2.3.0|...)

**Comment by anyj0527** [comment]
@jaeyun-jung Thanks for the review, I changed the code. And added a commit removing android/cpufeature dep. PTAL

**Review by jaeyun-jung — APPROVED** [review]

File: java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk [hunk 3]
```diff
@@ -133,7 +133,7 @@ LOCAL_SRC_FILES := $(NNSTREAMER_FILTER_TFLITE_SRCS)
 LOCAL_CXXFLAGS := -O3 -fPIC -frtti -fexceptions $(NNS_API_FLAGS) $(TFLITE_FLAGS)
 LOCAL_C_INCLUDES := $(TF_LITE_INCLUDES) $(NNSTREAMER_INCLUDES) $(GST_HEADERS_COMMON)
 LOCAL_EXPORT_LDLIBS := $(TFLITE_EXPORT_LDLIBS)
-LOCAL_WHOLE_STATIC_LIBRARIES := tensorflow-lite-lib cpufeatures
+LOCAL_WHOLE_STATIC_LIBRARIES := tensorflow-lite-lib
```
Comment (jaeyun-jung) [inline java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk:136]: Can you check below dependency later?
https://github.com/nnstreamer/nnstreamer/blob/42b797f74e5c901c3c471ae04b4c65b96a6f920d/jni/Android.mk#L237

**Comment by again4you** [comment]
LGTM!

File: java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk (hunk 3 above)
Comment (anyj0527) [reply java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk]: @jaeyun-jung Oops, I removed this commit (removing cpufeatures dep). I found that tflite < v1.15.0 requires `cpu-features.h`. PTAL.

**Review by myungjoo — APPROVED** [review]

**Pull request merged**